docs/source/09_development/02_set_up_pycharm.md

```diff
@@ -122,6 +122,33 @@ Click **OK** and then select **Remote Run** from the toolbar and click **Run** t
 
 To remotely debug, click the debugger button as [described above](#debugging).
 
+
+## Advanced: Docker interpreter
+
+```eval_rst
+.. note::  This section uses features supported in PyCharm Professional Edition only.
+```
+
+Firstly, add a Docker interpreter. Go to **Preferences | Project Interpreter** as above and proceed to add a new interpreter. Select **Docker Interpreter** and select the target Docker image:
+
+![](../meta/images/pycharm_docker_interpreter_1.png)
+
+Click **Next** and check that the required packages appear:
+
+![](../meta/images/pycharm_docker_interpreter_2.png)
+
+Click **Next** and wait for PyCharm to index your new Python interpreter.
+
+Click **OK**. Go to **Run / Debug Configurations** to add a **Python Run**. For the script path, get the path of the Kedro CLI on the remote computer by running `which kedro` (macOS / Linux) in the appropriate environment (/usr/local/bin/kedro in this case). Specify `run` as the parameter. Specify your working directory as the directory that contains your kedro project directory on your local machine. Optional: Edit the Docker container volume mount settings if it does not match the directory that contains your kedro project directory.   
```

Comment:
```suggestion
Click **OK**. Go to **Run / Debug Configurations** to add a **Python Run**. For the script path, get the path of the Kedro CLI on an instantiated image by running `which kedro` (macOS / Linux) in a container environment. Specify `run` as the parameter. Specify your working directory as the directory that contains your kedro project on your local machine. Optional: Edit the Docker container volume mount settings if it does not match the directory that contains your kedro project directory.   
```

Reply:
I assume _the Docker container volume mount settings_ refer to the field `Docker container settings`? Have you experience issues with it not auto-populating correctly?

Reply:
Yes pycharm does auto-populates it, but since I had to change it in the screenshot to remove my local path info, I wanter to avoid confusing people. But good point, I think making it optional is fine with me.